docs/kyma-environment-broker/03-03-runtime-operations.md

```diff
@@ -96,7 +96,7 @@ The upgrade process contains the following steps:
 
 ## Provide additional steps
 
-You can configure Runtime operations by providing additional steps. To add a new step, follow these tutorials:
+You can configure Runtime operations by providing additional steps. To add a new step, follow the following tutorials:
```

Comment:
```suggestion
You can configure Runtime operations by providing additional steps. To add a new step, follow the tutorials:
```

Reply:
Maybe `these` would be an even better choice?